<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>907</width>
    <height>538</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>张海山锐谐体</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MatchstickEquationSolver</string>
  </property>
  <widget class="QLabel" name="DB_label">
   <property name="geometry">
    <rect>
     <x>237</x>
     <y>278</y>
     <width>132</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>张海山锐谐体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>或选择右边的等式之一：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="DB_comboBox">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>279</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>3ds Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
        height: 25px;
        border-radius: 4px;
        border: 1px solid rgb(255, 255, 255);
        background: transparent;
}
QComboBox:enabled {
        color: rgb(255, 255, 255);
}
QComboBox:!enabled {
        color: rgb(155, 155, 155);
}
QComboBox::drop-down:hover {
        background: rgba(255, 255, 255, 30);
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="solveInput_pushButton">
   <property name="geometry">
    <rect>
     <x>447</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>3ds Light</family>
     <pointsize>12</pointsize>
     <weight>3</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 12pt &quot;3ds Light&quot;;color: rgb(255, 255, 255);
border-radius: 3px;
border: 1px solid rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Solve</string>
   </property>
  </widget>
  <widget class="QPushButton" name="solveDB_pushButton">
   <property name="geometry">
    <rect>
     <x>447</x>
     <y>337</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>3ds Light</family>
     <pointsize>12</pointsize>
     <weight>3</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 12pt &quot;3ds Light&quot;;color: rgb(255, 255, 255);
border-radius: 3px;
border: 1px solid rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Solve</string>
   </property>
  </widget>
  <widget class="QLabel" name="difficulty_label">
   <property name="geometry">
    <rect>
     <x>377</x>
     <y>397</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Difficulty：</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>317</x>
     <y>87</y>
     <width>306</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="matchNum_label">
      <property name="font">
       <font>
        <family>张海山锐谐体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>请选择可以移动的火柴棍数目：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="matchNum_comboBox">
      <property name="font">
       <font>
        <family>3ds Light</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox {
        height: 25px;
        border-radius: 4px;
        border: 1px solid rgb(255, 255, 255);
        background: transparent;
}
QComboBox:enabled {
        color: rgb(255, 255, 255);
}
QComboBox:!enabled {
        color: rgb(155, 155, 155);
}
QComboBox::drop-down:hover {
        background: rgba(255, 255, 255, 30);
}</string>
      </property>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="input_label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>176</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>张海山锐谐体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>请输入需要求解的等式：</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>170</y>
     <width>341</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QComboBox" name="fst_10_comboBox">
      <property name="font">
       <font>
        <family>3ds Light</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox {
        height: 25px;
        border-radius: 4px;
        border: 1px solid rgb(255, 255, 255);
        background: transparent;
}
QComboBox:enabled {
        color: rgb(255, 255, 255);
}
QComboBox:!enabled {
        color: rgb(155, 155, 155);
}
QComboBox::drop-down:hover {
        background: rgba(255, 255, 255, 30);
}</string>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="fst_1_comboBox">
      <property name="font">
       <font>
        <family>3ds Light</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox {
        height: 25px;
        border-radius: 4px;
        border: 1px solid rgb(255, 255, 255);
        background: transparent;
}
QComboBox:enabled {
        color: rgb(255, 255, 255);
}
QComboBox:!enabled {
        color: rgb(155, 155, 155);
}
QComboBox::drop-down:hover {
        background: rgba(255, 255, 255, 30);
}</string>
      </property>
      <item>
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="operator_comboBox">
      <property name="font">
       <font>
        <family>3ds Light</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox {
        height: 25px;
        border-radius: 4px;
        border: 1px solid rgb(255, 255, 255);
        background: transparent;
}
QComboBox:enabled {
        color: rgb(255, 255, 255);
}
QComboBox:!enabled {
        color: rgb(155, 155, 155);
}
QComboBox::drop-down:hover {
        background: rgba(255, 255, 255, 30);
}</string>
      </property>
      <item>
       <property name="text">
        <string>+</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>-</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>×</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="sec_10_comboBox">
      <property name="font">
       <font>
        <family>3ds Light</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox {
        height: 25px;
        border-radius: 4px;
        border: 1px solid rgb(255, 255, 255);
        background: transparent;
}
QComboBox:enabled {
        color: rgb(255, 255, 255);
}
QComboBox:!enabled {
        color: rgb(155, 155, 155);
}
QComboBox::drop-down:hover {
        background: rgba(255, 255, 255, 30);
}</string>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="sec_1_comboBox">
      <property name="font">
       <font>
        <family>3ds Light</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox {
        height: 25px;
        border-radius: 4px;
        border: 1px solid rgb(255, 255, 255);
        background: transparent;
}
QComboBox:enabled {
        color: rgb(255, 255, 255);
}
QComboBox:!enabled {
        color: rgb(155, 155, 155);
}
QComboBox::drop-down:hover {
        background: rgba(255, 255, 255, 30);
}</string>
      </property>
      <item>
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>3ds Light</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>   =</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="trd_10_comboBox">
      <property name="font">
       <font>
        <family>3ds Light</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox {
        height: 25px;
        border-radius: 4px;
        border: 1px solid rgb(255, 255, 255);
        background: transparent;
}
QComboBox:enabled {
        color: rgb(255, 255, 255);
}
QComboBox:!enabled {
        color: rgb(155, 155, 155);
}
QComboBox::drop-down:hover {
        background: rgba(255, 255, 255, 30);
}</string>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="trd_1_comboBox">
      <property name="font">
       <font>
        <family>3ds Light</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox {
        height: 25px;
        border-radius: 4px;
        border: 1px solid rgb(255, 255, 255);
        background: transparent;
}
QComboBox:enabled {
        color: rgb(255, 255, 255);
}
QComboBox:!enabled {
        color: rgb(155, 155, 155);
}
QComboBox::drop-down:hover {
        background: rgba(255, 255, 255, 30);
}</string>
      </property>
      <item>
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="quit_pushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>460</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>3ds Light</family>
     <pointsize>12</pointsize>
     <weight>3</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 12pt &quot;3ds Light&quot;;color: rgb(255, 255, 255);
border-radius: 3px;
border: 1px solid rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="about_pushButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>460</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>3ds Light</family>
     <pointsize>12</pointsize>
     <weight>3</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 12pt &quot;3ds Light&quot;;color: rgb(255, 255, 255);
border-radius: 3px;
border: 1px solid rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </widget>
  <widget class="QPushButton" name="regenerate_pushButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>281</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 12pt &quot;3ds Light&quot;;color: rgb(255, 255, 255);
border-radius: 3px;
border: 1px solid rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Regenerate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hide_pushButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>460</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 12pt &quot;3ds Light&quot;;color: rgb(255, 255, 255);
border-radius: 3px;
border: 1px solid rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Hide</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
